src/dataflow/src/render/reduce.rs

```diff
@@ -786,7 +786,10 @@ where
                 // never happen so for now just log an error if it does.
                 for (val, cnt) in input.iter() {
                     if cnt < &0 {
+                        #[cfg(not(debug_assertions))]
```

Comment:
i think we can just always log regardless of the panic? could be helpful if the logfile and the panic output go to two different places (idk if we do that)

Reply:
Logs are how we report panics, so no need to do that! https://github.com/MaterializeInc/materialize/blob/62ffac82b3f7be7c2da5b0e4e2e4d0d93475a389/src/materialized/src/bin/materialized/main.rs#L641-L655